### ISSUE TITLE
[NBug] Input string was not in a correct format.

### ISSUE THREAD

**Issue by JDECOMBE** [gitextensions/gitextensions#9071]
:warning: The sections below must be filled in and this text must be removed or the issue will be closed.

## Error Details
```
System.FormatException: Input string was not in a correct format.
   at System.Number.StringToNumber(String str, NumberStyles options, NumberBuffer& number, NumberFormatInfo info, Boolean parseDecimal)
   at System.Number.ParseInt32(String s, NumberStyles style, NumberFormatInfo info)
   at GitUI.Editor.Diff.DiffLineNumAnalyzer.Analyze(String diffContent)
   at GitUI.Editor.Diff.DiffViewerLineNumberControl.DisplayLineNumFor(String diff)
   at GitUI.Editor.FileViewerInternal.SetText(String text, Action openWithDifftool, Boolean isDiff)
   at GitUI.Editor.FileViewer.<>c__DisplayClass103_0.<ViewPatchAsync>b__0()
   at GitUI.Editor.FileViewer.ShowOrDeferAsync(Int64 contentLength, Func`1 showFunc)
   at GitUI.Editor.FileViewer.<ViewPatchAsync>d__103.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at GitUI.CommandsDialogs.RevisionDiffControl.<ShowSelectedFileDiffAsync>d__35.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at GitUI.CommandsDialogs.RevisionDiffControl.<<DiffText_ExtraDiffArgumentsChanged>b__39_0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at GitUI.ThreadHelper.<>c__DisplayClass13_0.<<FileAndForget>b__0>d.MoveNext()
```

## Additional information
No idea how it happens, I was litteraly away for my computer for 20 minutes, came back and when I focused the Git Extension window it crashed

## Environment
- Git Extensions 3.4.3.9999
- Build d4b0f48bbf3e39a71f5d7ff5231be5678d4aa0f1
- Git 2.21.0.windows.1 (recommended: 2.25.1 or later)
- Microsoft Windows NT 10.0.18363.0
- .NET Framework 4.8.4300.0
- DPI 96dpi (no scaling)

**Comment by gerhardol** [comment]
If this is caused in the same as #8634, it was fixed in #8661
Previous occurrences with explanation has been related to use of UTF-7 (that is removed in 3.5)

Duplicate of #7146

**Comment by RussKie** [comment]
Presume fixed. If not, please provide a standalone repro on the latest versions, and we'll reopen.
